src/test/java/seedu/address/testutil/StaffBuilder.java

```diff
@@ -116,6 +118,14 @@ public StaffBuilder withShiftDays(Set<Integer> shiftDays) {
         return this;
     }
 
+    /**
+     * Set the {@code Specialisation} of the {@code Staff} that we are building.
+     */
+    public StaffBuilder withSpecialisations(String ... specialisations) {
+        this.affiliations = SampleDataUtil.getAffiliationSet(specialisations);
+        return this;
+    }
```

Comment:
Could this have been added accidentally? Line 125 does not seem to match the intended purpose of this.

Anyways, I think having specialisation for the `StaffBuilder` might not be appropriate? Since it can potentially be sub-classed as a `Nurse`.

Reply:
Removed. I think i was experimenting making specialisation compulsory for all Doctors (including staff typed), so this would help build a staff/doctor. Mainly because Staff should sort of be an abstract class, so testing a Staff object is slightly more complex.